templates/scenarios/ts/default-bot-message-extension/teamsBot.ts

```diff
@@ -165,11 +165,23 @@ export class TeamsBot extends TeamsActivityHandler {
     query: AppBasedLinkQuery
   ): Promise<MessagingExtensionResponse> {
     const attachment = CardFactory.thumbnailCard("Image Preview Card", query.url, [query.url]);
+
+    // By default the link unfurling result is cached in Teams for 30 minutes.
+    // The code has set a cache policy and removed the cache for the app.
     return {
       composeExtension: {
         type: "result",
         attachmentLayout: "list",
         attachments: [attachment],
+        suggestedActions: {
+          actions: [
+            {
+              title: "default",
```

Comment:
Is this `title` needed?

Reply:
It is a required, not an optional input, only in ts.

Reply:
Indeed `title` is required in `CardAction` type. If the doc doesn't give any suggestions for the `title` value, "default" is ok to me.